tests/testthat/test_save_plot.R

```diff
@@ -0,0 +1,42 @@
+library(ggplot2)
+library(tibble)
+
+context('Save Plots')
+
+# Dummy templates
+foo <- ggplot2::ggplot()
+bar <- ggplot2::ggplot()
+dummy_templates <- list('foo'=foo, 'bar'=bar)
+performer_id <- "baz"
+
+# Displaylab templates
+templates <- load_templates()
+
+test_that("templates are saved in save_path directory with .png format", {
+  save_path <- getwd()
```

Comment:
This should be a temp directory that gets cleaned up after the tests regardless of test failure or error bailout.  We'll make that a new issue.